lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -191,6 +191,14 @@ div.course-wrapper {
         margin: 0 auto;
       }
     }
+    .disable-proctored-exam-btn {
```

Comment:
A suggestion here would be to use the base proctored-exam-btn class and add a state-based css class, following our naming conventions (based on https://smacss.com/book/type-state)  styling guide. (More detailed here: https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS#naming-syntax) 

To be more specific, the outcome in this case would be something like this: 

proctored-exam-btn {
   // base styles here

   // cases next, (such as &.success, &.failure, etc. as shown above.) 

   // state: is-disabled 
   &.is-disabled {
      // this would be where the style rules of ".disable-proctored-exam-btn" would go. 
   }
}